diagnostic.rb

```diff
@@ -2,15 +2,26 @@
 # Diagnostic answers to:
 
 # question 2:
-
+Licensee.create surname: 'Bishop',
+                        given_name: 'Caren',
+                        gender: 'f',
+                        height: 63,
+                        weight: 132 ,
+                        born_on: '1943-09-26',
+                        eye_color: 'Brown',
+                        hair_color: 'Black'
 # question 3:
-Licensee.transaction do
-  CSV.foreach 'data/licensees.csv', headers: true do |row|
+Licensee.transaction dosss
+  CSV.foreach(Rails.root + 'data/licensees.csv',
+      headers: true) do |row|
+        licensee = row.to_hash
+                      Licensee.create! licensee unless Licensee.exists? licensee
   end
 end
 
 # question 4:
-
-# question 5:
+Licensee.where eye_color: 'Hazel', hair_color: 'Brown', hair_color: 'Black'
```

Comment:
Not quite. `eye_color: 'Hazel', hair_color: 'Brown', hair_color: 'Black'` specifies a hash, and a hash's keys must be unique. You will only get either `hair_color: 'Brown'` _or_ `hair_color: 'Black'`. And having only one will not make the right query.